src/components/TaskListView/TaskListView.js

```diff
@@ -0,0 +1,35 @@
+import './TaskListView.css'
+import { useState } from 'react'
+import TableRow from '../TableRow/TableRow'
+
+
+const TaskListView = (props) => {
+
+  const [tasks, setTasks] = useState(props.savedData)
+
+  const deleteTask = (id) => {
+    const allOtherTasks = tasks.filter(saved => saved.id !== id)
+    setTasks(allOtherTasks)
+  }
+
+  const rows = tasks.map((saved, index) => {
+    return (<TableRow saved={saved} key={`row-${index}`} deleteTask={deleteTask} />)
+  }) 
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Category</th>
+          <th>Task</th>
+          <th className='complete-header'>Complete</th>
+        </tr>
+      </thead>
+      <tbody>
+        {!!tasks.length ? rows : <tr className='no-tasks'>Save a task to view it here!</tr>}
```

Comment:
We have `!!tasks.length?`. Would that be the same as `tasks.length?`
If so, can we please change it?

Reply:
Great question!! So just using `tasks.length` in this scenario is going to evaluate to `0` and truthy because the tasks array exists, so to be specific in the conditional I had to say `!!tasks.length` because I'm trying to have the conditional check for the truthy/falsey value of it.